docs/guide/events.md

```diff
@@ -32,7 +32,7 @@ If no listener is registered, incoming events are immediately ignored, preventin
     datasets: [{
       datalabels: {
         listeners: {
-          click: function(context) {
+          click: function(context, event) {
             // Receives `click` events only for labels of the first dataset.
             // The clicked label index is available in `context.dataIndex`.
             console.log('label ' + context.dataIndex + ' has been clicked!');
```

Comment:
We should demonstrate how to use `event` in this example (only). I'm not sure anymore about the ChartEvent API but I would suggest something like:

```js
console.log('label ' + context.dataIndex + ' has been clicked!');
console.log('mouse is at position x:', event.x, 'and y:', event.y);

if (event.native.ctrlKey) {
  console.log('control key is pressed!');
}
```

Reply:
oki!

Reply:
done